source/get-started/virtual-machine-install/hyper-v.rst

```diff
@@ -77,23 +83,22 @@ Create and configure new VM
    specifying the options below:
 
    - **Name**: Choose name (for example, ClearLinuxOS-VM)
-   - **Generation**: Generation 2
+   - **Specify Generation**: Generation 1
    - **Startup memory**: 2048 MB or more
+   - **Configure Networking**: Change :guilabel:`Connection` to `Default Switch`
    - **Connect Virtual Hard Disk**: Select :guilabel:`Use an existing virtual
      hard disk` and browse to find the :file:`clear-[VERSION]-hyperv.vhdx`
```

Comment:
This needs to be changed to `clear-[VERSION]-azure-hyperv.vhd`

Reply:
Got it. Good catch. Thanks.